src/name_service/delivery/update.clj

```diff
@@ -0,0 +1,25 @@
+(ns name-service.delivery.update
+  (:require [clojure.spec.alpha :as s]
+            [name-service.delivery.spec :as spec]
+            [name-service.delivery.state :as state]
+            [name-service.delivery.utils :as utils]
+            [cheshire.core :as json]))
+
+
+(defn- update-key-binding [json-params]
+  (let [kv-a (:update json-params)
+        kv-b (:where json-params)
+        key-a (-> kv-a utils/map-key-first)
+        val-a (-> kv-a utils/map-value-first)
+        key-b (-> kv-b utils/map-key-first)
+        val-b (-> kv-b utils/map-value-first)]
```

Comment:
Same as above `(utils/map-key-first kv-b)` should be enough no need for `->` macro.